docs/README.md

```diff
@@ -140,103 +144,103 @@ In order to define the Quickorder custom page UI, you must use the blocks export
 }
 ```
 
-4. Configure each one of the blocks previously declared as you wish using their props (check out the following tables). 
+4. Configure each one of the blocks previously declared as you wish using their props (check out the following tables).
 
 #### `store-list`
 
-| Prop name | Type | Description | Default value |
-| :-------: | :--: | :---------: | :-----------: |
-| `filterByTag` | `string` | Desired tags to filter the Pickup Points fetched and displayed. You can | `undefined` |
-| `icon` | `string` | Icon URL (`svg` or `png`) to be displayed alongside with the `store-list` blocks. | Google's default. |
-| `iconWidth` | `number` | Icon width in pixels (`px`). | Image default width. |
-| `iconHeight` | `number` | Icon height in pixels (`px`). | Image default height. |
+|   Prop name   |   Type   |                                    Description                                    |     Default value     |
+| :-----------: | :------: | :-------------------------------------------------------------------------------: | :-------------------: |
+| `filterByTag` | `string` |      Desired tags to filter the Pickup Points fetched and displayed. You can      |      `undefined`      |
+|    `icon`     | `string` | Icon URL (`svg` or `png`) to be displayed alongside with the `store-list` blocks. |   Google's default.   |
+|  `iconWidth`  | `number` |                           Icon width in pixels (`px`).                            | Image default width.  |
+| `iconHeight`  | `number` |                           Icon height in pixels (`px`).                           | Image default height. |
```

Comment:
No need to add the extra space :) The former version is better to read and edit the file, and it works just fine once rendered!